packages/framework/client-logger/app-insights-logger/src/fluidAppInsightsLogger.ts

```diff
@@ -276,40 +271,20 @@ export class FluidAppInsightsLogger implements ITelemetryBaseLogger {
 					uniqueFilterNamespaces.add(filter.namespacePattern);
 				}
 
-				const uniqueFilterNamespaceExceptions = new Set<string>();
 				for (const patternException of filter.namespacePatternExceptions ?? []) {
 					if (!patternException.startsWith(filter.namespacePattern)) {
 						throw new Error(
 							"Cannot have a namespace pattern exception that is not a child of the parent namespace",
 						);
 					}
-					if (uniqueFilterNamespaceExceptions.has(patternException)) {
-						throw new Error(
-							"Cannot have duplicate namespace pattern exception filters",
-						);
-					} else {
-						uniqueFilterNamespaceExceptions.add(patternException);
-					}
 				}
-				validatedFilters.push(filter);
 			} else if ("categories" in filter) {
 				// These are filters that only contain "categories".
-				for (const category of filter.categories) {
-					if (pureCategoryFilters.has(category)) {
-						console.warn("Redundant category filters found. Filters will be merged.");
-					}
-					pureCategoryFilters.add(category);
+				if (hasPureCategoryFilter) {
+					throw new Error("Cannot have multiple filters that only define categories");
```

Comment:
We can validate these in the same way that the others by adding an entry with empty namespace to `uniqueFilterNamespaces`, which also matches how we should think of them (filters for the default namespace).

Reply:
Updated to this.